scripts/generate_plugin_json.ts

```diff
@@ -28,7 +28,7 @@ if (!fs.existsSync(manifestFileDir)) {
 }
 
 const localPluginPaths = docsNavData
-  .find((plugin) => plugin.title === "Plugins")
+  .find((plugin) => plugin.title.includes("Plugins"))
```

Comment:
Because we lengthened "Plugins" titles to "Installing Grouparoo Plugins" and "Plugins Documentation", we have to change how this is generated.